app/webpack/stylesheets/open_studios_subdomain/_container_header.scss

```diff
@@ -12,7 +12,6 @@
   z-index: 50;
 }
 .container-header__brand {
-  position: absolute;
```

Comment:
I think we want to keep this.  Taking it out makes the header no longer centered on large screens.  If we need to take it out just for mobile, that might work.